pyproject.toml

```diff
@@ -17,6 +17,7 @@ python = "^3.7 || ^3.9"
 ifaddr = ">=0.1.0"
 requests = ">=2.0"
 six = ">=1.10.0"
+rstcheck = "^3.3.1"
```

Comment:
@pavoni I think this should be in `[tool.poetry.dev-dependencies]` section of the pyproject.toml?

Reply:
Thanks https://github.com/pavoni/pywemo/pull/187